viewer/templates/view.html.j2

```diff
@@ -10,6 +10,7 @@ var nno_csd = {{ nno_csd }};
 var last_csd = {{ last_csd }};
 var csd_list = {{ csd_list }};
 var opinions = {{ opinions }};
+var opinion_count = {{opinion_count}};
```

Comment:
Unneeded.  See the comment in `daily.js` below about setting `getElementById("opinion-count").innerHTML`.